src/tests/ftest/util/agent_utils.py

```diff
@@ -285,6 +287,9 @@ def start(self):
         # Copy certificates
         self.manager.job.copy_certificates(
             get_log_file("daosCA/certs"), self._hosts)
+        self.manager.job.copy_telemetry_certificates(
+            get_log_file("daosTelemetryCA"), self._hosts)
+        self.manager.job.generate_telemetry_certificates(self._hosts, "daos_agent")
```

Comment:
Maybe I'm missing something but how can we copy them before generating?

Reply:
So the certificate will be created on two stage first to create the private CA on Admin node and copy and create the individual certificate on server/client.
I see your point on naming confusion, let me change the function name to avoid confusion.